src/plugins/advanced_settings/public/management_app/components/field/field.tsx

```diff
@@ -453,6 +480,15 @@ export class Field extends PureComponent<FieldProps> {
           />
         </EuiText>
       );
+    } else if (setting.isPermissionControlled) {
+      return (
```

Comment:
Do we need to show this tip for dashboard admins to let them know this UI setting is only controlled by them?